imports/plugins/included/simple-inventory/server/no-meteor/mutations/updateSimpleInventoryBulk.js

```diff
@@ -0,0 +1,147 @@
+import SimpleSchema from "simpl-schema";
+import Random from "@reactioncommerce/random";
+import Logger from "@reactioncommerce/logger";
+import { ProductConfigurationSchema, SimpleInventoryCollectionSchema } from "../simpleSchemas";
+
+const logCtx = { name: "core-inventory" };
+
+const inputSchema = new SimpleSchema({
+  productConfiguration: ProductConfigurationSchema,
+  canBackorder: {
+    type: Boolean,
+    optional: true
+  },
+  inventoryInStock: {
+    type: SimpleSchema.Integer,
+    min: 0,
+    optional: true
+  },
+  isEnabled: {
+    type: Boolean,
+    optional: true
+  },
+  lowInventoryWarningThreshold: {
+    type: SimpleSchema.Integer,
+    min: 0,
+    optional: true
+  },
+  shopId: String
+});
+
+const updateFields = [
+  "canBackorder",
+  "inventoryInStock",
+  "isEnabled",
+  "lowInventoryWarningThreshold"
+];
+
+const defaultValues = {
+  canBackorder: false,
+  inventoryInStock: 0,
+  isEnabled: false,
+  lowInventoryWarningThreshold: 0
+};
+
+// eslint-disable-next-line require-jsdoc
```

Comment:
When you write function docs, include a warning that we aren't currently checking permissions in here because it's not exposed through GraphQL, but permissions should be added if that ever changes.